query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverterConfig.java

```diff
@@ -18,12 +18,16 @@ public class PinotFunctionConverterConfig {
   private static final String PERCENTILE_AGGREGATION_FUNCTION_CONFIG = "percentileAggFunction";
   private static final String DISTINCT_COUNT_AGGREGATION_FUNCTION_CONFIG =
       "distinctCountAggFunction";
+  private static final String DISTINCT_COUNT_MV_AGGREGATION_FUNCTION_CONFIG =
+      "distinctCountMvAggFunction";
```

Comment:
I did consider the approach of re-using existing configuration `distinctCountAggFunction`.

```
distinctCountAggFunction = DISTINCTCOUNTHLL
distinctCountAggOverrides = {
 "attribute_id_1" = DISTINCTCOUNT
}
```

We can consider above config as follow:
- for all single value attributes except `attribute_id_1`, use `DISTINCTCOUNTHLL`
- for all multi-value attributes use `DISTINCTCOUNTHLLMV`

However, the above approach will need internal mapping for corresponding multi-value function. In the above case:
```
multi-value-disctinct-count-func-map = {
"DISTINCTCOUNTHLL" : "DISTINCTCOUNTHLLMV"
}
```
So, for any new supported function, it will need a code change. So, thought of extending the config it self.